src/plugins/ui_core_views/filter_evaluation.ts

```diff
@@ -41,7 +40,7 @@ export class FilterEvaluationPlugin extends UIPlugin {
   hiddenRows: Set<number> = new Set();
   isEvaluationDirty = false;
 
-  allowDispatch(cmd: Command) {
```

Comment:
Note : I changed the allowDispatch interface of the current local plugins, but I couldn't find a way to type UI plugins correctly to prevent them from handling core commands.

We can do shit like that in ts, w/o typing error: 
![image](https://user-images.githubusercontent.com/31045223/221821803-2f9bd758-950b-47b4-9e61-483c70ca9821.png)

Reply:
You can't prevent that.

Nothing prevents a subclass to have a more general API, as long as it implements the parent's interface.

```ts
class A {
  coucou(a: number) {
    return true
  }
}
class B extends A {
  coucou(n: number | string) {
    return true
  }
}
```
Let's say I have `a: A;`  but `a` is actually an instance of `B` under the hood, but I don't know it. It's totally fine, I don't care that `a` can actually handle/do more stuff, I just use it as an `A`

However, if it doesn't have the same API than the parent, then it does work.
```ts
class B2 extends A {
  // error: Type 'number' is not assignable to type 'string'
  coucou(n: string) {
    return true
  }
}
```

Here since `LocalCommand` (parent API) is assignable to `LocalCommand | CoreCommand` (child class), everything is fine.

cc @anhe-odoo